src/cmap/connection.ts

```diff
@@ -341,6 +341,7 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
     }
 
     if (!operationDescription) {
```

Comment:
Instead of unsetting the the socket timeout in two places, can we

- always unset it at the top of the method
- conditionally reset it from monitor connections with more responses?

Reply:
updated